### PR TITLE
Add Horcrux volume plugin to plugins.md

### PR DESCRIPTION
**Pull request by muthu-r**
Signed-off-by: Muthukumar R muthur@gmail.com

### CODE REVIEW

**Comment by calavera** [comment]
LGTM

**Comment by muthu-r** [comment]
Thanks David.

File: docs/extend/plugins.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ The following plugins exist:
   another volume plugin that provides multi-host volumes management for Docker
   using GlusterFS.
 
+* The [Horcrux Volume Plugin](https://github.com/muthu-r/horcrux)
+  An On Demand, Version controlled access to your Data. Horcrux is an open-source plugin,
+  written in Go, and supports SCP, [Minio] (https://www.minio.io), and Amazon S3.
```
Comment (thaJeztah) [inline docs/extend/plugins.md]: Looks like the minio link has a space between `]` and `(`, making it not work as link.

File: docs/extend/plugins.md [hunk 2]
```diff
@@ -53,6 +53,10 @@ The following plugins exist:
   another volume plugin that provides multi-host volumes management for Docker
   using GlusterFS.
 
+* The [Horcrux Volume Plugin](https://github.com/muthu-r/horcrux)
+  An On Demand, Version controlled access to your Data. Horcrux is an open-source plugin,
```
Comment (thaJeztah) [inline docs/extend/plugins.md]: The "An On Demand" sentence is a bit weird, perhaps you can use the example I provided here; https://github.com/docker/docker/pull/20057#discussion_r52163446?

**Comment by muthu-r** [comment]
Sebastian,
Done.. Pls check!

Regards,
Muthu

**Comment by muthu-r** [comment]
Matched the wordings from your last comment.

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by thaJeztah** [comment]
ping @moxiegirl not sure if you're going to do cherry-picking for docs; if you do, I'll add a cherry-pick label

**Comment by muthu-r** [comment]
Thanks Sebastian.

**Pull request merged**